src/panels/lovelace/components/hui-generic-entity-row.ts

```diff
@@ -93,8 +93,9 @@ class HuiGenericEntityRow extends LitElement {
         ${hasSecondary
           ? html`
               <div class="secondary">
-                ${this.secondaryText ||
-                this.config.secondary_info === "entity-id"
+                ${this.secondaryText
+                  ? this.secondaryText
```

Comment:
this results in the same logic ?

Reply:
I thought so too, but the row was always showing entity-id. Check the demo site and you'll see it is currently showing the entity-id instead of info

Reply:
![image](https://user-images.githubusercontent.com/1287159/76378671-04dc4500-631c-11ea-87f9-fb07d3e8464c.png)